# default_scenario_name: 'Inspection'
# scenarios_data_folder: '/home/nao/ros_robocup_ws/data/scenarios'

# THIS FOLLOWING PATH SHOULD FIT WITH EVERY WORKSPACE
scenarios_data_folder: '../data'
# default_scenario_name: 'CleanUp2020CPEScenario'
default_scenario_name: ''